### DISCUSSION TITLE
Add ability to paste image in from Clipboard

### DISCUSSION THREAD

**Discussion by stayupthetree** [mealie-recipes/mealie#4762] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

Having to save an image I take a screenshot of in order to upload it via file browser

### Please provide a concise description of the feature that would resolve your issue.

The ability to paste in an image directly from the clipboard.

### Please consider and list out some caveats or tradeoffs made in your design decision

None that I am aware of, the ability to upload image should still remain, this just enhances it.

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by lionking102** [comment]
This would be especially useful for image import via AI. Would love this feature!
